types/alpinejs/alpinejs-tests.ts

```diff
@@ -471,10 +471,17 @@ import { reactive, effect, stop, toRaw } from '@vue/reactivity';
     }));
 
     // $ExpectType void
-    Alpine.data('dropdown', (initialOpenState = false) => ({
+    Alpine.data('dropdown', (initialOpenState: boolean = false) => ({
```

Comment:
I think the type should be inferred from the `= false` alone and the declaration should not be required
```suggestion
    Alpine.data('dropdown', (initialOpenState = false) => ({
```